tests/helpers/Courses/createCourse.test.js

```diff
@@ -0,0 +1,22 @@
+const createCourse = require("../../../helpers/Courses/createCourse");
+const createUser = require("../../../helpers/Users/createUser");
+const { Courses } = require("../../../db/Models");
+const dbHelpers = require("../../../db/databaseHelpers");
+
+beforeAll(async() => {
+  await dbHelpers.createTables();
+});
+
+afterAll(async() => {
+  await dbHelpers.wipeDBTables();
+});
+
+describe("The createCourse fn", () => {
+  it("should insert a course", async() => {
+    await createUser("1", "tutor@email.com", "1234", "tutor");
+    await createCourse("1", "algebra", 10, 10);
+    const course = await Courses.findOne({ where: { tutorId: "1", courseName: "algebra" } });
+    expect(course).toBeDefined();
```

Comment:
Test to see that the course values match up.

Reply:
4e141f20b15932e3806106aaba5843f5f3b2a076